gemd/entity/bounds/integer_bounds.py

```diff
@@ -1,34 +1,51 @@
 """Bounds an integer to be between two values."""
+from math import isfinite
+from typing import Union
+
 from gemd.entity.bounds.base_bounds import BaseBounds
 
-from typing import Union
+__all__ = ["IntegerBounds"]
 
 
 class IntegerBounds(BaseBounds, typ="integer_bounds"):
-    """
-    Bounded subset of the integers, parameterized by a lower and upper bound.
+    """Bounded subset of the integers, parameterized by a lower and upper bound."""
 
-    Parameters
-    ----------
-    lower_bound: int
-        Lower endpoint.
-    upper_bound: int
-        Upper endpoint.
+    def __init__(self, lower_bound, upper_bound):
+        self._lower_bound = None
+        self._upper_bound = None
```

Comment:
These should be assigned to the parameters to preserve backwards compatible behavior.

Reply:
Oops, missed the lines below. Nevermind.